open_city_profile/urls.py

```diff
@@ -48,7 +50,13 @@ def healthz(*args, **kwargs):
 
 
 def readiness(*args, **kwargs):
-    return HttpResponse(status=200)
+    response_json = {
+        "status": "ok",
+        "packageVersion": __version__,
+        "commitHash": settings.COMMIT_HASH,
+        "buildTime": settings.APP_BUILD_TIME.strftime("%Y-%m-%dT%H:%M:%S.000Z"),
+    }
+    return HttpResponse(json.dumps(response_json), status=200)
```

Comment:
Django also has `JsonResponse` so you could do:
```suggestion
    return JsonResponse(response_json, status=200)
```